common/src/db/multi_model.rs

```diff
@@ -0,0 +1,89 @@
+use sea_orm::{
+    ColumnTrait, DbErr, EntityTrait, FromQueryResult, IntoSimpleExpr, Iterable, QueryResult,
+    QuerySelect, Select, SelectModel, Selector,
+};
+use sea_query::{ColumnRef, SimpleExpr};
+
+pub trait ColumnsPrefixed: Sized {
+    fn try_columns_prefixed<C, I>(self, prefix: &str, cols: I) -> Result<Self, DbErr>
+    where
+        C: ColumnTrait,
+        I: IntoIterator<Item = C>;
+}
+
+impl<T: QuerySelect> ColumnsPrefixed for T {
+    fn try_columns_prefixed<C, I>(mut self, prefix: &str, cols: I) -> Result<Self, DbErr>
+    where
+        C: ColumnTrait,
+        I: IntoIterator<Item = C>,
+    {
+        for col in cols.into_iter() {
+            if let SimpleExpr::Column(col_ref) = col.into_simple_expr() {
+                match col_ref {
+                    ColumnRef::Column(name)
+                    | ColumnRef::TableColumn(_, name)
+                    | ColumnRef::SchemaTableColumn(_, _, name) => {
+                        let mut prefixed = String::new();
+                        prefixed.push_str(prefix);
+                        prefixed.push_str(&name.to_string());
+
```

Comment:
Turns out I hate `mut` more than macros... who knew? :woman_shrugging: 
```suggestion
                        let prefixed = format!("{prefix}{}", name.to_string());
```